net2/IntelTool.js

```diff
@@ -209,15 +209,27 @@ class IntelTool {
       fd = 'in';
     }
 
+    const ipCache = {};
     const _ipList = flowUtil.hashHost(ip) || [];
+    _ipList.forEach( (ele) => {
+      if(Array.isArray(ele)) {
```

Comment:
a little duplication with updateHashMapping, suggest refactor updateHashMapping to support both ip and domain